tests/performance-jmh/README.md

```diff
@@ -0,0 +1,41 @@
+Apache Artemix JMH Benchmarks
+-------
+This module contains optional [JMH](http://openjdk.java.net/projects/code-tools/jmh/) performance tests.
+
+Note that this module is an optional part of the overall project build and does not deploy anything, due to its use
+of JMH which is not permissively licensed. The module must either be built directly.
```

Comment:
"The module must either be built directly." Sentence fragment that doesnt make any sense after the 'or enabled with a profile' followup was dropped from the original proton-j source inspiration.